tests/unit/ssg-module/test_id_translate.py

```diff
@@ -21,3 +26,172 @@ def test__split_namespace():
     ns, n = sn("{}emptynamespace")
     assert not ns
     assert n == "emptynamespace"
+
+
+def test_tagname_to_abbrev():
```

Comment:
Wouldn't it be better to define a mapping `tagname->expected abbreviation, and just iterate over the pairs?